sumologic/sumologic_monitors_library_monitor.go

```diff
@@ -196,4 +197,9 @@ type WebhookNotificiation struct {
 	PayloadOverride string `json:"payloadOverride,omitempty"`
 }
 
+type BurnRate struct {
+	BurnRateThreshold float64 `json:"burnRateThreshold,omitempty"`
+	TimeRange         string  `json:"timeRange"`
```

Comment:
why not optional with `omitempty` ?

Reply:
Not sure what you mean by that? But shouldn't this be required as this is not top level. We have the same required constraint in the schema as well

Reply:
Sorry I meant both `BurnRateThreshold ` and `TimeRange` should have same behaviour. So shouldn't we remove `omitemtpy` from `BurnRateThreshold` ?

Reply:
Before this change, timeRange was always required for all trigger conditions. Now it is optional at the top level (since burn rate condition does not require it). Inside this struct however, both are required. hence I think we should change top level field to include `omitempty` and remove it from here in burn rate threshold

Reply:
removed